api/apps/api/src/modules/scenarios/scenario.service.ts

```diff
@@ -0,0 +1,117 @@
+import { HttpService, Injectable } from '@nestjs/common';
+import { FetchSpecification } from 'nestjs-base-service';
+
+import { AppInfoDTO } from '@marxan-api/dto/info.dto';
+import { AppConfig } from '@marxan-api/utils/config.utils';
+import { ScenarioFeaturesService } from '@marxan-api/modules/scenarios-features';
+import { AdjustPlanningUnits } from '@marxan-api/modules/analysis/entry-points/adjust-planning-units';
+import { apiGlobalPrefixes } from '@marxan-api/api.config';
+
+import { CostSurfaceFacade } from './cost-surface/cost-surface.facade';
+import { ScenariosCrudService } from './scenarios-crud.service';
+import { JobStatus } from './scenario.api.entity';
+
+import { CreateScenarioDTO } from './dto/create.scenario.dto';
+import { UpdateScenarioDTO } from './dto/update.scenario.dto';
+import { UpdateScenarioPlanningUnitLockStatusDto } from './dto/update-scenario-planning-unit-lock-status.dto';
+
+@Injectable()
+export class ScenarioService {
```

Comment:
I suggest to stick to `<Plural>Service` (as for the recently-refactored `ProjectsService`), and `<Plural>CrudService` (this is already ok in this PR for `ScenariosCrudService`). Filename to be adjusted accordingly here.

Reply:
@hotzevzl I actually forgot to mention this but I have done this "miss" on purpose - to have a clean diff (otherwise git would mess with diff merging new/old content).

Reply:
(as you already reviewed, will change it in this PR)

Reply:
https://github.com/Vizzuality/marxan-cloud/pull/241/commits/8e4e9a68d948000c3dda981102d5afb8a224884d